data/recipes.js

```diff
@@ -5381,4 +5381,10 @@ let recipesData = [
       ]
   }
 ]
+
+const recipesData = recipeData.map(recipe => {
+    recipe.hits = 0;
+    return recipe;
+})
```

Comment:
friggin love this hahaha, just a note - in the job, we'd need to run a migration to add this property to each recipe in our database - fun!

Reply:
ohhhh! I had just mapped over the original data set to add the property starting at 0 lol